lib/components/work-order-list/work-order-list.tsx

```diff
@@ -35,8 +70,29 @@ export const WorkOrderList = ({ assetId }: WorkOrderListProps) => {
     );
   }
 
+  const getStatusLabel = (statusCode: WorkOrdersFilters) => {
+    return REPAIR_FILTER_LIST.find((item) => item.code === statusCode)?.value;
```

Comment:
Can you add a default here. Doubt it's reachable but would prefer not to see a `null` in the ui